modules/gateways/razorpay.php

```diff
@@ -2,9 +2,11 @@
 function razorpay_config() {
 
     $configarray = array(
-        "FriendlyName" => array("Type" => "System", "Value" => "Razorpay (Credit Card/Debit Card/Net Banking)"),
+        "FriendlyName" => array("Type" => "System", "Value" => "Razorpay"),
         "KeyId" => array("FriendlyName" => "Key Id", "Type" => "text", "Size" => "50", "Description" => "Enter your Razorpay Key Id here",),
         "KeySecret" => array("FriendlyName" => "Key Secret", "Type" => "text", "Size" => "50", "Description" => "Enter your Razorpay Key Secret here",),
+        "ThemeLogo" => array("FriendlyName" => "Logo URL", "Type" => "text", "Size" => "50", "Description" => "Enter the full URL for the logo including 'https://' or 'http://' [max: 56x56 px | file: png/jpg/gif/ico]",),
```

Comment:
The logo image must be 128x128 or larger. It has to be hosted on an HTTPS website, since its embedded inside a secure iframe.